front/app/containers/Admin/projects/project/participationContext/components/VotingInputs.tsx

```diff
@@ -80,18 +82,40 @@ export default ({
   ideas_order,
   handleIdeaDefaultSortMethodChange,
   handleAllowAnonymousParticipationOnChange,
-}: Props) => {
+  handleVotingMethodOnChange,
+}: VotingInputsProps) => {
+  const { projectId } = useParams() as {
+    projectId: string;
+  };
+  const props = {
```

Comment:
Putting everything into a single "props" variable allows us to pass it around in the VoteMethodConfig without using long lists of names/attributes. For example, the specific inputs for different voting methods will use different attributes. Passing all props in a "props" attribute into the config makes it simple to make sure all functions have what they need.